src/Map/src/Map.php

```diff
@@ -20,27 +20,34 @@
  */
 final class Map
 {
+    private Markers $markers;
+    private Polygons $polygons;
+    private Polylines $polylines;
+
     public function __construct(
         private readonly ?string $rendererName = null,
         private ?MapOptionsInterface $options = null,
         private ?Point $center = null,
         private ?float $zoom = null,
         private bool $fitBoundsToMarkers = false,
-        /**
-         * @var array<Marker>
+        /*
+         * @var Marker[]
          */
-        private array $markers = [],
+        array $markers = [],
 
-        /**
-         * @var array<Polygon>
+        /*
+         * @var Polygon[]
          */
-        private array $polygons = [],
+        array $polygons = [],
 
-        /**
-         * @var array<Polyline>
+        /*
```

Comment:
```suggestion
        /**
```

Reply:
agree but I don't know why fapbot want to remove one * ...

Reply:
I think it's because the PHPDoc must be put on __construct(), not on the parameter directly